### PR TITLE
Add a formula for elm-format.

### PR DESCRIPTION
**Pull request by valrus**
https://github.com/avh4/elm-format is a nice tool for enforcing the [Elm](http://elm-lang.org) style guide but it's still in alpha.

### CODE REVIEW

File: Formula/elm-format.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  depends_on "ghc" => :build
+  depends_on "cabal-install" => :build
+
+  def install
+    install_cabal_package
+  end
+
+  test do
+    (testpath/"Test.elm").write(%(
```
Comment (DomT4) [inline Formula/elm-format.rb]: Can you use `<<-EOS.undent` here and then indent it? Looks neater than having one part of the formula glued to the left of the file.

**Comment by valrus** [comment]
OK, I made the requested change.

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by bfontaine** [comment]
We just merged #41 which added the 0.2.0-alpha version. Thank you anyway!

**Pull request closed without merging**